package.json

```diff
@@ -1,21 +1,16 @@
 {
   "name": "tc-ui-toolkit",
-  "version": "1.1.5",
+  "version": "1.1.6",
   "description": "React components used to develop tools for the desktop app translationCore",
-  "main": "dist/bundle.js",
+  "main": "lib/index.js",
   "display": "library",
   "scripts": {
     "lint": "eslint ./src",
     "lint:fix": "eslint ./src --fix",
     "test": "eslint ./src && jest",
-    "prestart": "npm run webpack:d",
-    "start": "npm run render-ui",
-    "render-ui": "webpack && cd tc-ui-toolkit-test && npm start",
```

Comment:
Why are you removing npm start ? We no longer need the UI tester?

Reply:
Oops, good point - it may actually work now.